packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx

```diff
@@ -46,6 +46,7 @@ const PatientChart: React.FC = () => {
             <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
             <aside>
               <PatientBanner {...state} />
+              <ExtensionSlot state={state} extensionSlotName="top-of-all-patient-dashboards-slot" />
```

Comment:
Extensions in slots can already be ordered. What's the advantage of adding a new slot here vs ensuring the extension is just loaded in position 0 of the next slot?

Reply:
I think that the `top-of-all-patient-dashboards-slot` was created to bring out the `What's new bar (flags summary)` which should be exactly after the `Patient-header`.  Not knowing about extension position functionality, i decided to create this extension here than creating it in the `patient-dashboard` like it was done before, which made the Vitals summary come first violating the designs.  
<img width="1206" alt="Screenshot 2023-01-17 at 09 02 26" src="https://user-images.githubusercontent.com/58003327/212823919-78c215a0-f3e8-45e9-adb0-f6be897d0387.png">
<img width="834" alt="Screenshot 2023-01-17 at 09 08 25" src="https://user-images.githubusercontent.com/58003327/212823932-aa2f7c57-6819-4ee4-a56c-56d0a8b61d69.png">

hope this answers https://github.com/openmrs/openmrs-esm-patient-chart/pull/926#discussion_r1071276118